src/main/java/org/generationcp/middleware/manager/WorkbenchDaoFactory.java

```diff
@@ -0,0 +1,108 @@
+package org.generationcp.middleware.manager;
+
+import org.generationcp.middleware.dao.CropPersonDAO;
+import org.generationcp.middleware.dao.CropTypeDAO;
+import org.generationcp.middleware.dao.PersonDAO;
+import org.generationcp.middleware.dao.ProjectActivityDAO;
+import org.generationcp.middleware.dao.ProjectDAO;
+import org.generationcp.middleware.dao.ProjectUserInfoDAO;
+import org.generationcp.middleware.dao.RoleDAO;
+import org.generationcp.middleware.dao.StandardPresetDAO;
+import org.generationcp.middleware.dao.ToolDAO;
+import org.generationcp.middleware.dao.UserInfoDAO;
+import org.generationcp.middleware.dao.WorkbenchSidebarCategoryDAO;
+import org.generationcp.middleware.dao.WorkbenchSidebarCategoryLinkDAO;
+import org.generationcp.middleware.dao.WorkbenchUserDAO;
+import org.generationcp.middleware.hibernate.HibernateSessionProvider;
+
+public class WorkbenchDaoFactory {
```

Comment:
We'll need to migrate some methods from the DaoFactory here. I guess it's less confusing to have them here, you can identify quickly on which db you are working on